help/destinations/catalog/personalization/pega-v2.md

```diff
@@ -0,0 +1,137 @@
+
+See [Activate audience data to streaming profile export destinations](../../ui/activate-streaming-profile-destinations.md) for instructions on activating audiences to this destination.
+
+### Destination attributes {#attributes}
+
+In the [[!UICONTROL Select attributes]](../../ui/activate-streaming-profile-destinations.md#select-attributes) step, Adobe recommends that you select a unique identifier from your [union schema](../../../profile/home.md#profile-fragments-and-union-schemas). Select the unique identifier and any other XDM fields that you want to export to the destination.
+
+### Mapping example: activating profile updates in [!DNL Pega Customer Decision Hub] {#mapping-example}
+
+Below is an example of correct identity mapping when exporting profiles to [!DNL Pega Customer Decision Hub].
+
+Selecting source fields:
+
+* Select an identifier (For example: CustomerID) as source identity that uniquely identifies a profile in Adobe Experience Platform and [!DNL Pega Customer Decision Hub].
+* Select XDM source profile attribute changes that need to be exported and updated in [!DNL Pega Customer Decision Hub].
+
+Selecting target fields:
+
+* Select the `CustomerID` namespace as target identity.
+* Select destination profile attribute names that need to be mapped to corresponding XDM source profile attributes.
+
+![Identity mapping](../../assets/catalog/personalization/pega/pega-source-destination-mapping.png)
+
+## Exported data / Validate data export {#exported-data}
+
+A successful audience membership update for a profile would insert the audience identifier, name and statuses in the Pega marketing audience membership datastore. The membership data is associated to a customer using Customer Profile Designer in [!DNL Pega Customer Decision Hub], as shown below.
+![Image of the UI screen where you can associate Adobe Audience Membership data to Customer, using Customer Profile Designer](../../assets/catalog/personalization/pega/pega-profile-designer-associate.png)
+
+The audience membership data is used in Pega Next-Best-Action Designer Engagement polices for next-best-action decisioning, as shown below.
+![Image of the UI screen where you can add Audience membership fields as conditions in Engagement Policies of Pega Next-Best-Action Designer](../../assets/catalog/personalization/pega/pega-profile-designer-engagment.png)
+
+The customer audience membership data fields are added as predictors in Adaptive models, as shown below.
+![Image of the UI screen where you can add Audience membership fields as predicators in Adaptive models, using Prediction Studio](../../assets/catalog/personalization/pega/pega-profile-designer-adaptivemodel.png)
+
+## Additional resources {#additional-resources}
```

Comment:
Refer to the following [!DNL Pega] documentation for more information:

-  [Setting up an OAuth 2.0 client registration](https://docs.pega.com/bundle/platform/page/platform/security/configure-oauth-2-client-registration.html)
- [Creating a real-time run for data flows](https://docs.pega.com/bundle/platform/page/platform/decision-management/data-flow-run-real-time-create.html)
- [Manage customer records in Customer Profile Designer](https://docs.pega.com/bundle/customer-decision-hub/page/customer-decision-hub/implement/profile-designer-data-management.html)

Reply:
Updated